docs/sources/setup-grafana/installation/debian/index.md

```diff
@@ -9,21 +9,23 @@ weight: 100
 
 # Install on Debian or Ubuntu
 
-This page explains how to install Grafana dependencies, download and install Grafana, get the service up and running on your Debian or Ubuntu system, and also describes the installation package details.
+This topic explains how to install Grafana dependencies, install Grafana on Linux Debian or Ubuntu, get the service up and running on your Debian or Ubuntu system, and describes the contents of the installation package.
 
-## Repository migration (November 8th 2022)
+You can install Grafana using the Grafana Labs APT repository, by downloading a `.deb` package, or by downloading a binary `.tar.gz` file.
```

Comment:
```suggestion
You can install Grafana using the Grafana Labs APT repository, by downloading a `.deb` package, or by downloading a binary `.tar.gz` file.

If you install via the `.deb` package or `.tar.gz` file, then you will need to manually update Grafana for each new version.
```
We might want to keep the APT repo at the top here as that is the preferred install mechanism, and ideally we want to make that more prominent on the download page as well.

Reply:
ok, cool. I'll make that section up.